SocialFilter/sections/specific.txt

```diff
@@ -425,6 +425,8 @@ russianemirates.com##.socialink_block
 elesquiu.com##.inner-share-bar
 asia.nikkei.com##.article__share-slot
 freesvg.org##.footer-social-big
+japan.cnet.com##.social-item-bottom
+japan.cnet.com##.leaf-social-inner
```

Comment:
This rule seems to cover both, however, we don't block print button etc. https://github.com/AdguardTeam/AdguardFilters/pull/103914#discussion_r773113029
I think in this case it's probably better to go with a generic filter `##a[href^="https://note.mu/intent/post"]`.

Reply:
@zloyden @Sergey-Lyapin

Reply:
@Yuki2718 you were right, I did the changes directly.